src/js/views/patients/patient/dashboard/dashboard_views.js

```diff
@@ -130,10 +130,8 @@ const ActionItemView = View.extend({
       return;
     }
 
-    const program = this.model.getProgram();
```

Comment:
these all shouldn't be removed

Reply:
Not sure where these came from. I think I messed up a rebase/merge. But I removed those changes.